source/system/String.ooc

```diff
@@ -311,8 +311,9 @@ strArrayFromCString: func ~hack (argc: Int, argv: String*) -> String[] {
 }
 cStringPtrToStringPtr: func (cstr: CString*, len: Int) -> String* {
 	toRet: String* = calloc(len, Pointer size) // otherwise the pointers are stack-allocated
-	for (i in 0 .. len) {
+	for (i in 0 .. len)
 		toRet[i] = makeStringLiteral(cstr[i], cstr[i] length())
-	}
 	toRet
 }
+
+//GlobalCleanup register(|| String free~all())
```

Comment:
@sebastianbaginski `String free~all()` seems to always crash with memory errors no matter where/how I call it, including like this.